examples/docker-rootful.yaml

```
@@ -62,8 +110,15 @@ probes:
       echo >&2 "docker is not installed yet"
       exit 1
     fi
-    if ! timeout 30s bash -c "until pgrep dockerd; do sleep 3; done"; then
-      echo >&2 "dockerd is not running"
+    if [ "{{.Param.Rootful}}" = "true" ]; then
+      target=dockerd
+      target_description="dockerd"
```

Minor nit, but why is one value quoted and the other one isn't? Given that the `else` block needs quoting for the second value, maybe quote everything?

```suggestion
      target="dockerd"
      target_description="dockerd"
```